extras/cloudshell/tutorial.md

```diff
@@ -0,0 +1,122 @@
+# Bank of Anthos Minikube Quickstart
+
+## 
+This tutorial shows you how to deploy the Bank of Anthos app to a Kubernetes cluster.
+
+You'll be able to run Bank of Anthos on:
+- a local **[minikube](https://minikube.sigs.k8s.io/docs/)** cluster, which comes built in to the Cloud Shell instance
+- a **[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)** cluster using a new or existing [Google Cloud Platform project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)
+
+Let's get started!
+
+## Kubernetes cluster setup
+
+Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+
+---
+### Minikube
+
+Minikube creates a local Kubernetes cluster on Cloud Shell.
+
+1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">Minikube Stopped</walkthrough-editor-spotlight> on the status bar.
```

Comment:
@kelsk  Should we add an image of the `status bar` here? Or is it something quiet straightforward?

Reply:
@Shabirmean good idea! I'll update the tutorial accordingly.